build.sh

```diff
@@ -138,12 +141,19 @@ build_v8() {
 	# The build actually requires that you clone using this git wrapper tool
 	cd ${wrkdir}
 	OLDPATH=${PATH}
-	PATH=${wrkdir}/depot_tools:${PATH}
-	fetch v8 || exit $?
+	# v8's build needs python 2.7.10; as we've already built that, we might
+	# as well use it rather than forcing the user to install their own.
+	PATH=${wrkdir}/cpython-inst/bin:${wrkdir}/depot_tools:${PATH}
+	# XXX we should check for errors when fetching, but currently that
+	# causes problems because fetch runs a script which aborts on OpenBSD
```

Comment:
The main VM builds fine; it's just a test that fails. So if you run "make" twice on OpenBSD, everything is fine. I'd prefer not to suppress this, so that we're reminded to check to see if future v8 versions fix the bug in question. [I must admit, it's unclear to me from the google code issue as to whether the v8 people might look into this themselves or not.]